imagesize.py

```diff
@@ -177,14 +176,18 @@ def get(filepath):
                 raise ValueError("Invalid TIFF file: width and/or height IDS entries are missing.")
         # handle SVGs
         elif size >= 5 and head.startswith(b'<?xml'):
```

Comment:
Please replace with `elif size >= 5 and head.startswith(b'<?xml') or head.startswith(b'<svg'):` otherwise files without an XML declaration will not be handled.

Reply:
Sounds reasonable to me. I will add this.